Makefile

```diff
@@ -75,7 +75,7 @@ install: crd rbac
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 .PHONY: deploy
-deploy: generate manifests
+deploy: install generate manifests
```

Comment:
shouldn't you call manifests before generate, since the result of manifests can affect the generated bindata.go?